lib/taskana-core/src/main/java/pro/taskana/impl/TaskServiceImpl.java

```diff
@@ -731,7 +731,7 @@ private void standardSettings(TaskImpl task, Classification classification,
         }
     }
 
-    List<TaskSummary> augmentTaskSummariesByContainedSummaries(List<TaskSummaryImpl> taskSummaries) {
+    List<TaskSummary> augmentTaskSummariesByContainedSummaries(List<TaskSummaryImpl> taskSummaries, TaskQueryImpl taskQuery) {
```

Comment:
I haven't thought about this case. But I think we don't need this. The requirement is rather "Give me a tasks where there is an email attachment". We should wait for a more precise requirment on how to handle multiple attachments in a list (as we already discussed)